app/models/location.rb

```diff
@@ -12,6 +19,10 @@ def admin_url
     RailsAdmin::Engine.routes.url_helpers.edit_path(self.class.to_s.downcase, id)
   end
 
+  def attributes
```

Comment:
this needs to be included with where the admin_url is merged into attributes, instead of a duplicate method definition